lib/resources/dstu2/appointment.yaml

```diff
@@ -21,13 +21,18 @@ fields:
   description: The type of appointment that is being booked.
   action: read
   example: |
-             {"type": {
-                "coding": [
-                  {"system": "http://snomed.info/sct",
-                   "code": "394581000",
-                   "display": "Community medicine",
-                   "userSelected": false}],
-                "text": "Same Day"}
+             {
+               "type": {
+                 "coding": [
+                   {
+                     "system": "http://snomed.info/sct",
+                     "code": "394581000",
+                     "display": "Community medicine",
+                     "userSelected": false
+                   }
+                 ],
+                "text": "Same Day"
```

Comment:
I think this needs one more closing squiggly bracket

Reply:
Addressed with f54f352a457572e639496cf1f96efa62816cd041